war/src/main/webapp/help/system-config/master-slave/jnlp-tunnel.html

```diff
@@ -3,8 +3,8 @@
     to establish a communication channel.
     But some security sensitive network can prevent you from making this connection.
     This can also happen when Jenkins runs behind a load balancer,
-    <a href="http://www.serverwatch.com/tutorials/article.php/3290851">apache reverse proxy</a>
-    into <a href="http://en.wikipedia.org/wiki/Demilitarized_zone_(computing)">DMZ</a>, and so on.
+    <a href="https://www.serverwatch.com/guides/reverse-proxying-with-apache-2-0/">apache reverse proxy</a>
```

Comment:
Probably better to link to our own docs?

```suggestion
    <a href="https://www.jenkins.io/doc/book/system-administration/reverse-proxy-configuration-apache/">apache reverse proxy</a>
```